sources/models/src/aws-k8s-1.15/mod.rs

```diff
@@ -3,7 +3,7 @@ use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
```

Comment:
We'll need the change and settings for the aws-ecs-1 model as well.  And I'd argue for aws-dev, too.

Reply:
Addressed (and tested) in [f18d841](https://github.com/bottlerocket-os/bottlerocket/compare/ec0abbf943fe3806294f53fd4e118be3d1f529fc..f18d841ee1c2b35d5f09136ea3f7136e2a4d65ae)